plugins/outputs/prometheus_client/prometheus_client.go

```diff
@@ -14,23 +15,30 @@ import (
 
 var invalidNameCharRE = regexp.MustCompile(`[^a-zA-Z0-9_]`)
 
+type MetricWithExpiration struct {
+	Metric     prometheus.Metric
+	Expiration time.Time
+}
+
 type PrometheusClient struct {
-	Listen string
+	Listen             string
+	ExpirationInterval int64 `toml:"expiration_interval"`
```

Comment:
change this type to `internal.Duration`, that way it can reprepsented like `expiration_interval = "10s"`